README.md

```diff
@@ -128,7 +128,7 @@ job('upstreamJob') {
     }
 
     triggers {
-        githubPullRequest {
+        pullRequest {
```

Comment:
Hi! I have same error @vdice mentioned above with both methods. Which version do you use? My setup has GitHub Pull Request Builder 1.34.0 and Job DSL 1.57